gpustat/cli.py

```diff
@@ -96,12 +96,12 @@ def main(*argv):
     parser.add_argument(
         '-e', '--show-codec', nargs='?', const='enc,dec', default='',
         choices=['', 'enc', 'dec', 'enc,dec'],
-        help='Show encoder/decoder utilization'
+        help='Show encoder and/or decoder utilization'
     )
     parser.add_argument(
-        '-P', '--show-power', nargs='?', const='draw,limit',
-        choices=['', 'draw', 'limit', 'draw,limit', 'limit,draw'],
-        help='Show GPU power usage or draw (and/or limit)'
```

Comment:
It was never and/or limit. If `limit` is included, then `draw` is also always included.

Reply:
Do you mean passing `limit` and `draw, limit` gives the same output?

Reply:
Yes (`limit,draw` as well, but I removed that in this PR)